extra_data/karabo-bridge-stack-files

```diff
@@ -0,0 +1,61 @@
+#!/gpfs/exfel/sw/software/xfel_anaconda3/1.1/bin/python
```

Comment:
Can you remove this file and the extra README?

Reply:
Yes I will remove this, you already suggested that some time ago ...

Reply:
don't forget this :wink:

Reply:
Doh, my bad ... but now finally removed, together with the tentative README_file_Stack.md